data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -52,7 +53,7 @@ spec:
             privileged: true
           volumeMounts:
             - name: cnibin
-              mountPath: /opt/cni/bin
+              mountPath: {{ .CNIBinDir }}
       volumes:
```

Comment:
this is not needed, the volume serves as an abstraction. all scripts can be static the only place for tempting is from where we mount this directory

Reply:
I think I fixed it. tell me if not :)

Reply:
I do not understand @phoracek here.
It sounded to me like he was talking about having the scripts as static and the source of the mount to be templated. But I am unsure really. To me it sounds like we need all to be templates or use a variable, in order not to mess things up.

Reply:
I think I made a mess, but fixed it now..